iscdhcp/files/dhcpd.hosts

```diff
@@ -12,7 +12,9 @@
 host {{ h.name }} {
   hardware ethernet {{ h.mac }};
   fixed-address {{ h.ipaddr }};
+  {%- if h.get('insert-ddns-hostname', True) %} 
```

Comment:
i set this to True to keep default behaviour, because ddns-hostname is always set. 
if i set this to False wouldn't all ddns-hostname's deleted out of existing configs?
or am I wrong?